app/src/shared/contexts/ModalPortal/index.jsx

```diff
@@ -1,7 +1,20 @@
 // @flow
 
-import React, { type Node } from 'react'
-import Context from '$shared/contexts/Modal'
+import React, { type Node, type Context as ReactContext } from 'react'
+
+type Context = {
+    isModalOpen: boolean,
+    registerModal: ?() => void,
+    unregisterModal: ?() => void,
+}
+
+const defaultContext: Context = {
+    isModalOpen: false,
+    registerModal: null,
+    unregisterModal: null,
```

Comment:
really should avoid using `null` by default due to it breaking ability to set default arguments in functions